server/db/db.js

```diff
@@ -1,6 +1,8 @@
 const Sequelize = require('sequelize')
+const connectionString = require('../../config')
```

Comment:
It would be better if we pulled `connectionString` off of the import, instead of it _being_ the import. Otherwise we may have to refactor in the future if we want to add more properties to `config`. In other words, `config` should export an object with a `connectionString` property.